core/src/syn/parser/stmt/define.rs

```diff
@@ -1213,6 +1217,113 @@ impl Parser<'_> {
 		Ok(res)
 	}
 
+	pub fn parse_define_config(&mut self) -> ParseResult<DefineConfigStatement> {
+		let (if_not_exists, overwrite) = if self.eat(t!("IF")) {
+			expected!(self, t!("NOT"));
+			expected!(self, t!("EXISTS"));
+			(true, false)
+		} else if self.eat(t!("OVERWRITE")) {
+			(false, true)
+		} else {
+			(false, false)
+		};
+
+		let next = self.next();
+		let inner = match next.kind {
+			t!("GRAPHQL") => self.parse_graphql_config().map(ConfigInner::GraphQL)?,
+			_ => unexpected!(self, next, "a type of config"),
+		};
+
+		Ok(DefineConfigStatement {
+			inner,
+			if_not_exists,
+			overwrite,
+		})
+	}
+
+	fn parse_graphql_config(&mut self) -> ParseResult<GraphQLConfig> {
+		use graphql::{FunctionsConfig, TablesConfig};
+		let mut tmp_tables = Option::<TablesConfig>::None;
+		let mut tmp_fncs = Option::<FunctionsConfig>::None;
+		loop {
+			match self.peek_kind() {
+				t!("NONE") => {
+					self.pop_peek();
+					tmp_tables = Some(TablesConfig::None);
+					tmp_fncs = Some(FunctionsConfig::None);
+				}
+				t!("AUTO") => {
+					self.pop_peek();
+					tmp_tables = Some(TablesConfig::Auto);
+					tmp_fncs = Some(FunctionsConfig::Auto);
+				}
+				t!("TABLES") => {
+					self.pop_peek();
+
+					let next = self.next();
+					match next.kind {
+						t!("INCLUDE") => {
+							tmp_tables =
+								Some(TablesConfig::Include(self.parse_graphql_table_configs()?))
+						}
+						t!("EXCLUDE") => {
+							tmp_tables =
+								Some(TablesConfig::Include(self.parse_graphql_table_configs()?))
+						}
+						t!("NONE") => {
+							tmp_tables = Some(TablesConfig::None);
+						}
+						t!("AUTO") => {
+							tmp_tables = Some(TablesConfig::Auto);
+						}
+						_ => unexpected!(self, next, "`NONE`, `AUTO`, `INCLUDE` or `EXCLUDE`"),
+					}
+				}
+				t!("FUNCTIONS") => {
+					self.pop_peek();
+
+					let next = self.next();
+					match next.kind {
+						t!("INCLUDE") => {}
+						t!("EXCLUDE") => {}
```

Comment:
These keywords don't seem to do anything, is that intended?

Reply:
functions are not currently supported, I'll add it in the pr for that